lib/templates/plugin.js

```diff
@@ -20,17 +20,11 @@ import { <%= options.treeShake[lib.key].join(', ') %> } from '<%= lib.location %
 import options from './options'
 
 Vue.use(Vuetify, {
-<%
-if (options.treeShake) {
-  for (const lib of libImports) {
-    if (options.treeShake && options.treeShake[lib.key] && options.treeShake[lib.key].length > 0) {
-    %>
-    <%= lib.key %>: { <%= options.treeShake[lib.key].join(', ') %> }
-    <%
-    }
-  }
-}
-%>
+<% if (options.treeShake) { %>
+<%= libImports.filter(lib => Array.isArray(options.treeShake[lib.key]) && options.treeShake[lib.key].length > 0)
```

Comment:
if it isnt array it will just silently ignored and not working for user for unknown apparent reason for user

Reply:
@aldarund Is right, you just need to check that `options.treeShake[lib.key]` exists.